net-libs/mbedtls/mbedtls-2.6.0.recipe

```diff
@@ -0,0 +1,118 @@
+SUMMARY="An open source, portable, easy to use, readable and flexible SSL library"
+DESCRIPTION="mbed TLS (formerly known as PolarSSL) makes it trivially easy for \
+developers to include cryptographic and SSL/TLS capabilities in their (embedded) \
+products, facilitating this functionality with a minimal coding footprint."
+HOMEPAGE="https://tls.mbed.org"
```

Comment:
If you could add a / after the `.org` that would be cool ;-)
(Some browsers hide them, but real URIs have a trailing slash when there is no path after.)